changelog/984.feature.rst

```diff
@@ -0,0 +1,3 @@
+Add a ``symbol`` attribute to `~plasmapy.particles.particle_class.AbstractParticle`
+and therefore `~plasmapy.particles.particle_class.CustomParticle` and
+`~plasmapy.particles.particle_class.DimensionlessParticle`.
```

Comment:
Since the `breaking` change log should be removed (we're not breaking yet), update this change log to include info about the `Particle.particle` & `Particle.symbol` updates.

If you want to mention the `FutureWarning` for the deprecation, then the entry might be best suited as a `trivial` change log entry.  Maybe a `removal` change log entry, but I still think that is more suitable for when the feature is actually deleted.

Reply:
This all makes sense.  I wasn't quite sure how to best organize the change logs.

Reply:
The [towncrier docs](https://towncrier.readthedocs.io/en/actual-freaking-docs/quickstart.html#creating-news-fragments) say that `removal` signifies "a deprecation or removal of public API" so `removal` does seem most appropriate.